### DISCUSSION TITLE
Auth facade not working

### DISCUSSION THREAD

**Discussion by infinitetrinity** [spatie/laravel-multitenancy#420] [Q&A]
I have a single login page where user enter their domain and credential.
However I'm having problem with the Auth facade not checking against the Tenant even after making it current.

```php
$account = Account::firstWhere('domain', $domain);
$account->makeCurrent();
Auth::attempt($this->only('email', 'password')) //doesn't work as it still check against Landlord instead of Tenant.
```

Appreciate all the helps.

**Comment by denizgolbas** [comment]
Do it with guards.
